project/Dependencies.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2022 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import sbt._
+
+object Dependencies {
+
+
+  private def coomonDependencies(scalaVersion: String): Seq[ModuleID] = Seq(
+    "org.scala-lang"      %  "scala-compiler"               % scalaVersion,
```

Comment:
never had to add scala-compiler to the sbt dependencies. Why is it needed?

Reply:
Probably could be removed here